website/blog/2018-05-29-jest-23-blazing-fast-delightful-testing.md

```diff
@@ -0,0 +1,124 @@
+
+## Watch Mode Plugins
+
+We have completely rewritten the watch mode system to allow adding custom plugins to watch mode. Watch Mode Plugins now make it possible to hook into Jest events and provide custom menu options in the Watch Mode Menu. All of the default Watch Mode prompts are implemented as plugins in this system, and the docs to create your own are available [here](https://facebook.github.io/jest/docs/en/watch-plugins.html).
+
+With this change, we're also now able to bring back typeahead support as a Watch Mode Plugin via [](https://github.com/rogeliog/jest-plugin-filename)[jest-plugin-filename](https://github.com/jest-community/jest-plugin-filename)!
+
+![Typeahead plugin in action](/jest/img/blog/23-typeahead.gif)
+
+See [jest-plugin-filename](https://github.com/jest-community/jest-plugin-filename) for documentation and install instructions. Huge thank you to [@rogeliog](https://twitter.com/rogeliog) for the new watch mode plugin system and the jest-plugin-filename plugin!
+
+## Breaking Changes
+
+As with every major release, we are making a few breaking changes to make larger changes in the future possible and to push the testing experience to a new level. Here's a list of the biggest changes you may see:
+
+* **Require test descriptions**: We're now failing tests that do not include a description.
+* **Remove undefined props from React snapshots:** Smaller snapshots and proper React behavior.
+* **Remove deprecations**: We've removed mapCoverage since it's no longer needed. Additionally, we removed `jest.genMockFunction` and `jest.genMockFn` since these are the same `jest.fn`.
+* **Add snapshot names to failures: **We added the snapshot name (if provided) to the snapshot failure message so it's easier to find the snapshot that's failing.
+* **Replace mock timestamps**: We replaced mock timestamps with invocationCallOrder since two or mocks may often have the same timestamp, making it impossible to test the call order.
+
+## Other Improvements
+
+* **Watch mode coverage**: Coverage is now limited to only the files tested in watch mode or when using `—onlyChanged` and `—findRelatedTests`.
+* **Version documentation:** We added docs for each minor release back to Jest 22, and have removed all of the “Requires Jest X.X+” from the docs.
+* **Better snapshot summaries: **We overhauled the Snapshot Summary output to make obsolete snapshots more informative.
+* **Better stack traces**: We added stack traces to asynchronous errors, timeout errors, expect.assertions, and thrown non-errors. We're also indicating the column in the code frame!
+* **Better React 16 support**: Adds snapshot support for `React.Fragment`, `React.forwardRef`\*\* \*\*and `React.createContext`.
+
+## Jest Summit
+
+Last week, the Jest Core Team met for the Jest Summit at Facebook London where worked on and released Jest 23, announced the Jest Open Collective, and gave a number of talks:
```

Comment:
mention they'll be posted soon:tm:?

Reply:
the videos, that is

Reply:
Hoping to update this with the links before go-live